src/Avalonia.Visuals/Gpu/EGL.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Avalonia.Gpu
+{
+    /// <summary>
+    /// EGL wrapper class. Only small part of API is exposed currently.
+    /// https://www.khronos.org/registry/EGL
+    /// </summary>
+    public static class EGL
```

Comment:
EGL is the glue behind the underlying platform Windowing system and the OpenGL API. It handles context and surface creation. While it is platform independent, it would tie Avalonia to at least knowing about GL. IMO it should probably live in an Avalonia.OpenGL project that backends can (optionally) reference.